molgenis-core-ui/src/main/javascript/modules/react-components/AggregateTable.js

```diff
@@ -12,7 +14,7 @@
 	 * @memberOf component
 	 */
 	var AggregateTable = React.createClass({
-		mixins: [molgenis.ui.mixin.DeepPureRenderMixin],
+		mixins: [DeepPureRenderMixin],
```

Comment:
Can these mixins not be pulled up to superclass (it seems we need them almost everywhere)